04-django/01-python-intro/random_password.py

```diff
@@ -0,0 +1,10 @@
+import random
+import string
+length =random.randint(8, 12)
+choices= string.ascii_lowercase + string.ascii_uppercase + string.digits
+print(choices)
+def randomPassword(n):
+    list = [random.choice(choices) for i in range(n)]
```

Comment:
Be careful with naming your variable `list`, it will shadow the built in function with the same name.